### PR TITLE
A bit more documentation (fixes #24)

### LINKED ISSUE #24: Document how the different job scripts relate to each other

### PR DESCRIPTION
**Pull request by leplatrem**
fixes #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,18 @@
...
 
-## Web UI
+This repository has two main folders:
 
-The Web UI is written in Elm.
+* [ui/](https://github.com/mozilla-services/buildhub/tree/master/ui#readme): A Web UI to browse the available data
+* [jobs/](https://github.com/mozilla-services/buildhub/tree/master/jobs#readme): A set of jobs in charge of aggregating and keeping the data up-to-date.
 
-### Setting up the development environment
+![](overview.png)
```
Comment (glasserc) [inline README.md]: What's `AUS` again?

File: jobs/README.rst [hunk 2]
```diff
@@ -1,35 +1,45 @@
...
+    The ``user:pass`` in the command-line examples is the Basic auth for Kinto.
 
 
-Pulse listener
-==============
+Scrape archives
+===============
 
-Listen to Pulse build and publishes records on a ``builds`` collection.
+Scrape releases on https://archives.mozilla.org and publishes records on a ``archives`` collection.
```
Comment (glasserc) [inline jobs/README.rst]: Doesn't it also scrape nightlies, or are those considered releases too? Can it really be considered scraping if we're parsing JSON files that expose metadata? Can we describe a little bit about what information is available from this source? (`This source provides information about release and nightly versions, including their target platform, locale, channel, and version, as well as the URL where they are available for download and (for en-US builds) the tree and revision they were built from.`)

File: jobs/README.rst [hunk 3]
```diff
@@ -1,35 +1,45 @@
...
+    python3 scrape_archives.py --server http://localhost:8888/v1 --auth user:pass --debug
 
 
-Scrape archives
-===============
+System-Addons updates
+=====================
 
-Scrape releases on https://archives.mozilla.org and publishes records on a ``archives`` collection.
+Fetch information about available system addons updates for every Firefox release.
```
Comment (glasserc) [inline jobs/README.rst]: I only just realized that these are not system addons (per the field name), but rather system addons updates. Maybe something like `Add information to each `archive` record detailing the system addons for that release. Some are built in, but others are updated from AUS.` Maybe also `Each addon has its ID, a builtin version (if any), and an update available from AUS.`

File: README.md (hunk 1 above)
Comment (mostlygeek) [inline README.md]: automatic update service (balrog)

**Comment by leplatrem** [comment]
Final r?

**Comment by leplatrem** [comment]
I'll merge this because a storm of changes will come in #20

**Pull request merged**